locales/zh-cn/dpbGajO1GT8.srt

```diff
@@ -1,79 +1,79 @@
-﻿1
+1
 00:00:00,230 --> 00:00:02,680
-答案是 这是一个错误
+答案选择程序是错误的 (Error)
 
 2
 00:00:02,680 --> 00:00:06,290
-原因是第一个赋值语句中右边是 minutes
+原因是第一个赋值语句右侧有 minutes 这个变量
 
 3
 00:00:06,290 --> 00:00:09,840
-但是我们之前没有定义过 minutes
+但是我们在前面没有定义过它
 
 4
 00:00:09,840 --> 00:00:12,880
-所以如果我们使用了没有定义过的变量
+如果我们使用了未被定义的变量
 
 5
 00:00:12,880 --> 00:00:15,670
-那么它就不会有有意义的值
+Python 解释器就会报错 
 
 6
 00:00:15,670 --> 00:00:17,900
-Python 解释器就会显示错误
+因为解释器不知道这个变量的值
 
 7
 00:00:17,900 --> 00:00:21,990
-我们在 Python 解释器中试一下
+我们在 Python 解释器中演示一下这种错误情况
 
 8
 00:00:21,990 --> 00:00:25,350
-现在我们来看第一个赋值 当我们运行它的时候
+我们来看第一条赋值语句 当运行该语句时
 
 9
 00:00:25,350 --> 00:00:29,550
-显示名称 minutes 未定义的错误 我们之前还没有引入这个变量
+解释器会告诉我们 minutes 这个变量没有被定义 
 
 10
 00:00:29,550 --> 00:00:33,490
-所以 minutes 未被定义
+这是因为我们还没有引入这个变量 所以 minutes 这时还是未被定义的

```

Comment:
Minutes is not defined because we didn't introduce that variable yet. It looks like we're defining it. We've gotten in on the left side, but the right side needs to execute first. So without knowing the value of minutes,

10-13 这几句话，目前校对后的文字连起来看意思是对的，但是放到视频中的时候，会有与视频内容接不上的感觉，可以再斟酌一下！

Reply:
这里老师的意思也有一部分是重复的，为了让读起来流畅一些，我上次把中文翻译过来的句内顺序稍微调换了一下。